### PR TITLE
Add Company Name

### PR DESCRIPTION
**Pull request by xyan12345**

### CODE REVIEW

File: client/components/game/company-name/companyName.jsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+export default React.createClass({
+  render() {
+    return (
+      <div className="company_name">
```
Comment (idmontie) [inline client/components/game/company-name/companyName.jsx]: The class name should be `company-name`.

File: client/components/game/company-name/companyName.jsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
```
Comment (idmontie) [inline client/components/game/company-name/companyName.jsx]: Change the file name to `company-name.jsx`

File: client/components/game/company-name/companyName.jsx [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+export default React.createClass({
+  render() {
+    return (
+      <div className="company_name">
+        <h>{this.props.CompanyName}</h>
```
Comment (idmontie) [inline client/components/game/company-name/companyName.jsx]: Change the prop to `companyName`.

File: client/index.jsx [hunk 4]
```diff
@@ -3,6 +3,7 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import {default as Game} from './containers/game/game.jsx';
 
+
```
Comment (idmontie) [inline client/index.jsx:11]: Why is this extra line here?

File: client/components/game/company-name/companyName.jsx (hunk 3 above)
Comment (xyan12345) [inline client/components/game/company-name/companyName.jsx]: Do i just call
<h>{this.companyName}</h>?
Is that how to fix it?

Comment (idmontie) [inline client/components/game/company-name/companyName.jsx]: `{this.props.companyName}`

File: client/actions/actions.js [hunk 5]
```diff
@@ -3,27 +3,38 @@ export const ACTION_ENUM = {
...
   INTRO_SCENE: 'INTRO_SCENE',
   CREDITS_SCENE: 'CREDITS_SCENE',
+
 };
 
 export function newGame() {
   return { type: ACTION_ENUM.NEW_GAME };
 }
 
+<<<<<<< HEAD
```
Comment (idmontie) [inline client/actions/actions.js]: Fix this

File: client/components/game/program-list-item/program-list-item.jsx [hunk 6]
```diff
@@ -6,7 +6,12 @@ import { selectProgram } from '../../../actions/actions';
 export default React.createClass({
   onClick( e ) {
     e.preventDefault();
+<<<<<<< HEAD
```
Comment (idmontie) [inline client/components/game/program-list-item/program-list-item.jsx]: Fix merge

File: client/components/game/program-list-item/program-list-item.jsx [hunk 7]
```diff
@@ -6,7 +6,10 @@ import { selectProgram } from '../../../actions/actions';
 export default React.createClass({
   onClick( e ) {
     e.preventDefault();
+    dispatch( selectProgram(this.props.programName, this.props.credits));
```
Comment (idmontie) [inline client/components/game/program-list-item/program-list-item.jsx]: What happened here?

**Comment by idmontie** [comment]
This looks good except for the program-list-item onClick action

**Pull request merged**